app/mailers/issue_mailer.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 class IssueMailer < ApplicationMailer
+  include IssuesHelper
```

Comment:
Warum / Wozu das include des Helpers? Um ihn in der View zu verwenden sollte es reichen in anzugeben

Reply:
Das habe ich entfernt, da es nicht notwendig ist!